.travis.yml

```diff
@@ -2,9 +2,9 @@ language: node_js
 sudo: false
 
 node_js:
-  - '5'
-  - '4.2.0'
-  - '4'
+  - 8
```

Comment:
Any reason we're skipping node 9?

Reply:
I took it [from hapijs](https://github.com/hapijs/hapi/blob/master/.travis.yml), I guess they only officially support LTS